portfolio/src/main/java/com/google/sps/data/ShortestTaskFirst.java

```diff
@@ -36,7 +32,7 @@
    * This method schedules tasks from shortest to longest and returns a ScheduledTask Collection
    * based on the tasks that were able to be scheduled.
    */
-  public static Collection<ScheduledTask> shortestTaskFirst(
+  public Collection<ScheduledTask> schedule(
```

Comment:
Just a reminder for us that calling the getEmptyTimeRanges will look a bit different since it's in another class now

Reply:
I fixed it now rather than later to prevent us from forgetting.